psi4/src/psi4/libmints/mintshelper.cc

```diff
@@ -841,10 +835,12 @@ SharedMatrix MintsHelper::ao_shell_getter(const std::string &label, std::shared_
     ints->compute_shell(M, N, P, Q);
     const double *buffer = ints->buffer();
 
-    for (int m = 0, index = 0; m < mfxn; m++) {
+#pragma omp parallel for collapse(4) num_threads(nthread_)
```

Comment:
This loop is just doing assignment. I doubt it is CPU bound. Parallellization might actually hurt performance.

Reply:
I understand, I've reverted the code back in all instances this comment is made.